scripts/bin/create.py

```diff
@@ -222,11 +229,16 @@ def push_repo(project_name, default_branch):
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Template Creation Tool - Create an new open service(s)... fast!')
     parser.add_argument('REPOSITORY', type=str, help='The name of the service you want to create')
-    parser.add_argument('--type', choices=['FRONTEND', 'BACKEND', 'LIBRARY'], help='Sets the type of repository to be either a Play template for FRONTEND or BACKEND microservice or a Play library')
-    parser.add_argument('--github-token', help='The github token authorised to push to the repository')
-    parser.add_argument('--github', action='store_true', help='Does the repository already exists on github? Set --github for this repo to be cloned, and updated')
-    parser.add_argument('--with-mongo', action='store_true', help='Does your service require Mongo? This only available if the repository is of type "BACKEND"')
-    parser.add_argument('--default-branch', default='master', help='The default branch name for the GitHub repository when pushing changes')
+    parser.add_argument('--type', choices=['FRONTEND', 'BACKEND', 'LIBRARY'],
+                        help='Sets the type of repository to be either a Play template for FRONTEND or BACKEND microservice or a Play library')
+    parser.add_argument('--dry-run', action='store_true',
+                        help='Set --dry-run to avoid connecting with remote GitHub repository')
+    parser.add_argument('--github', action='store_true', help='deprecated')
+    parser.add_argument('--with-mongo', action='store_true',
+                        help='Does your service require Mongo? This only available if the repository is of type "BACKEND"')
+    parser.add_argument('--github-token', help='The GitHub token authorised to push to the repository')
+    parser.add_argument('--default-branch', default='master',
+                        help='The default branch name for the GitHub repository when pushing changes')
     args = parser.parse_args()
 
-    create_service(args.REPOSITORY, args.type, args.github, args.with_mongo, args.github_token, args.default_branch)
+    create_project(args.REPOSITORY, args.type, args.dry_run, args.with_mongo, args.github_token, args.default_branch)
```

Comment:
does the deprecated `args.github` need to be a fallback when `args.dry_run` is not set?

Reply:
Previously `--github` had to be present to sync with github but now this behaviour is on my default. B&D clone directly from github so can't remove `--github` just yet.